### ISSUE TITLE
A11y_A11y_.NETOnlineREPL_Benchmark_.NET Editor_Keyboard:Loss of keyboard focus is found after the editor section.

### ISSUE THREAD

**Issue by Charanrajkb** [dotnet/try#789]
**Environment details:**
https://docs.microsoft.com/
Browser: Chromium Edge Version 80.0.361.50 (Official build) (64-bit)
OS Details:Windows 10 

GitHubTags:#A11yTCS;#A11ySev2;#A11y_.NETOnlineREPL_Benchmark;#.NET Online REPL;#Benchmark;#DesktopWeb;#Win10;#MAS2.4.7;#Accessibility;#.NET Online REPL;#keyboard;#ChromiumEdge;#A11yMAS;

​**Repro Steps:​**
1.Hit the URL "https://docs.microsoft.com/" and login with appropriate credentials.
2.Tab till "Enter focus Mode" button and press enter.
3.Navigate using the tab key in the editor section.
4.Observe the keyboard focus after the editor section. 

**Actual Result:​**
When navigated using the tab key after the editor there is loss of keyboard focus is observed.

**Expected Result:​**
There should be no loss of keyboard focus after the editor section.

**User Impact:​**
Keyboard and low vision users will find difficulty in identifying the keyboard focus after the editor section.

**Attachment:**
[789_A11y_.NETOnlineREPL_.NET Editor_Keyboard.pptx](https://github.com/dotnet/try/files/4207892/789_A11y_.NETOnlineREPL_.NET.Editor_Keyboard.pptx)

**Comment by LadyNaggaga** [comment]
@Charanrajkb This is an external bug and I have notified the docs team[ here ](https://github.com/MicrosoftDocs/feedback/issues/2531)

**Comment by keerthiduvvuri** [comment]
#Closed;
